simulations/vip-diamond-comptroller-testnet/simulations.ts

```diff
@@ -0,0 +1,446 @@
+import { impersonateAccount } from "@nomicfoundation/hardhat-network-helpers";
+import { expect } from "chai";
+import { Contract, Signer } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { forking, pretendExecutingVip, testVip } from "../../src/vip-framework";
+import { vipDiamondTestnet } from "../../vips/vip-Diamond-comptroller-testnet";
+import Comptroller from "./abi/Comptroller.json";
+import IERC20Upgradeable from "./abi/IERC20UpgradableAbi.json";
+import VBEP20_DELEGATE_ABI from "./abi/VBep20DelegateAbi.json";
+
+const UNITROLLER = "0x94d1820b2D1c7c7452A163983Dc888CEC546b77D";
+const DIAMOND = "0xB2243Da976F2cbAAa4dd1a76BF7F6EFbe22c4CFc";
+
+const Owner = "0xce10739590001705F7FF231611ba4A48B2820327";
+const zeroAddr = "0x0000000000000000000000000000000000000000";
+const VBUSD = "0x08e0A5575De71037aE36AbfAfb516595fE68e5e4";
+const VUSDT = "0xb7526572FFE56AB9D7489838Bf2E18e3323b441A";
+
+const initMainnetUser = async (user: string) => {
+  await impersonateAccount(user);
+  return ethers.getSigner(user);
+};
```

Comment:
Maybe we could reuse the existing one?

```suggestion
import { initMainnetUser } from "../../src/utils";
```